app/views/session/new.html.erb

```diff
@@ -0,0 +1,18 @@
+<h1>Sign In</h1>
+
+<div>
+  <div>
+    <%= form_for(:session, url: signin_path) do |f| %>
```

Comment:
I suggest to use gem 'simple_form' for any forms you do. SImpleForm looks like too big and complex gem but it has many features we need in real projects.